### PR TITLE
Add support for client cert auth

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

### CODE REVIEW

File: pulpcore/cli/common/__init__.py [hunk 1]
```diff
@@ -42,6 +42,10 @@ def _config_callback(ctx: click.Context, param: Any, value: Optional[str]) -> No
 @click.option("--base-url", default="https://localhost", help="Api base url")
 @click.option("--username", help="Username on pulp server")
 @click.option("--password", help="Password on pulp server")
+@click.option("--client-cert", help="Path to client certificate")
+@click.option(
+    "--client-key", help="Path to client private key. Not required if client-cert contains this."
+)
```
Comment (daviddavis) [inline pulpcore/cli/common/__init__.py:48]: I'm tempted to use `--cert` and `--key` here since that's the options curl uses.

**Comment by ianballou** [comment]
Working for me with Katello!

File: README.md [hunk 2]
```diff
@@ -65,6 +65,19 @@ login admin
 password password
 ```
 
+### Katello
```
Comment (ianballou) [inline README.md:68]: Works for me! Tested with a Katello development box.

File: pulpcore/cli/common/__init__.py (hunk 1 above)
Comment (mdellweg) [reply pulpcore/cli/common/__init__.py:48]: Would work for me.

Comment (mdellweg) [reply pulpcore/cli/common/__init__.py:48]: Now you missed those two appearances.

**Review by mdellweg — APPROVED** [review]
:cake:

**Review by mdellweg — APPROVED** [review]

**Pull request merged**